bin/test-cleanup

```diff
@@ -28,3 +28,9 @@ if ! clusterroles=$(kubectl get clusterroles -oname | grep -E "/linkerd-$linkerd
 else
   kubectl delete $clusterroles
 fi
+
+if ! crd=$(kubectl get crd/serviceprofiles.linkerd.io -oname); then
```

Comment:
The service profile CRD is global to the whole cluster, in that all linkerd installs in the cluster rely on the same CRD being present. If I run this cleanup script in a cluster where I have linkerd installed in a different, non-test namespace, that linkerd install will break, which seems problematic. Would you mind reverting this change?

Reply:
@klingerf Sure will revert. Should this motivate that we someone scope and/or name the ServiceProfile CRD to a specific Linkerd control-plane, similar to ClusterRoles?

Reply:
Ideally yes, but in practice we can't change the name of our CRDs without also changing the name/group of the ServiceProfile resource itself, since Kubernetes requires that CRDs be named as follows:

> must be spec.names.plural+"."+spec.group

So a name like `linkerd-l5d-integration-serviceprofiles.linkerd.io` is invalid.

Reply:
😞